lessons/lesson-template.md

```diff
@@ -0,0 +1,74 @@
+- title: 
+- tags: 
+
+###Standards:
```

Comment:
Put pre-work up top, highlighted appropriately, where students can find it easily?